lib/rubocop/cop/layout/space_inside_block_braces.rb

```diff
@@ -82,7 +82,7 @@ class SpaceInsideBlockBraces < Cop
         include RangeHelp
 
         def on_block(node)
-          return if node.keywords?
+          return if node.keywords? || node.body.nil? && node.multiline?
```

Comment:
I'd put this in a separate `return` and a remark why this branch exists.

Reply:
Yeah, I updated it!